Modules/AuctionManager/AuctionManager.lua

```diff
@@ -198,26 +198,31 @@ function AuctionManager:StartAuction(itemId, itemLink, itemSlot, baseValue, maxV
     self.allowNegativeStandings = configuration:Get("allowNegativeStandings")
     -- Auctioning
     -- Start Auction Messages
-    local auctionMessage = string.format(CLM.L["Auction of %s"], itemLink)
-    if note:len() > 0 then
-        auctionMessage = auctionMessage .. " (" .. tostring(note) .. ")"
-    end
-    self.note = note
-    -- Max 2 raid warnings are displayed at the same time
-    SendChatMessage(auctionMessage , "RAID_WARNING")
-    auctionMessage = ""
-    if baseValue > 0 then
-        auctionMessage = auctionMessage .. string.format(CLM.L["Minimum bid: %s."], tostring(baseValue))
-    end
-    if maxValue > 0 then
-        auctionMessage = auctionMessage .. string.format(CLM.L["Maximum bid: %s."], tostring(maxValue))
-    end
-    auctionMessage = auctionMessage .. string.format(CLM.L["Auction time: %s."], tostring(auctionTime))
-    self.antiSnipe = configuration:Get("antiSnipe")
-    if self.antiSnipe > 0 then
-        auctionMessage = auctionMessage .. string.format(CLM.L["Anti-snipe time: %s."], tostring(self.antiSnipe))
+    if CLM.GlobalConfigs:GetAuctionWarning() then
```

Comment:
This seems it may lead to not setting of `self.antiSnipe` (line 217) at all. Did you check the behavior with `antiSnipe > 0` and triggering it when there is no warnings?